masterlist.yaml

```diff
@@ -112,9 +238,69 @@ globals:
         text: '[최신 LOOT 스레드](%1%).'
       - lang: zh_CN
         text: '[最新的LOOT版本](%1%)。'
+      - lang: da
+        text: '[Seneste LOOT-tråd](%1%).'
       - lang: ja
         text: '[LOOTの最新スレッドを開く](%1%)（英語）'
     subs: [ 'https://loot.github.io/latest-thread/' ]
+# SKYRIM SE Latest
+  - type: say
+    condition: 'file("../SkyrimSE.exe") and version("../SkyrimSE.exe", "1.0.0.0", <)'
```

Comment:
Given that the executable version doesn't reflect the game version, I think it's pointless checking it. You could check the CRC instead, but I don't know if different regions get different executables (which would all have different CRCs)...